Week_7.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/*
+
+Microsoft Interview Coaching Pilot 2018 (MICP 2018)
+Author: Aishna Gupta
+Language: JAVA
+
+************WEEK 6************
+
+Problem description: Design a vending machine.
+Clarify: No more restrictions, just design a general vending machine we are familiar with.
+*/
+
+public class Week_7 {
+	
+	
+	public class Item{
+		private String Item_Id;
+		private String Item_Name;
+		private int Item_Quantity;
+		private int Item_Price;
```

Comment:
Item_Quantity is probably a property of a vending machine, not the item itself.  Does a candy bar know how many other candy bars are in the machine?  This may also apply to Item_Price if two vending machines want to sell the same item at a different price

Reply:
Another note, is price really an integer?  What if something costs $1.25